README.md

```diff
@@ -123,18 +122,25 @@ Some mathematical set relations will be useful throughout this specification.
 We shall denote:
 
 * The equivalence of two sets `A` and `B` as `A = B`
-* The intersection as `A ∩ B`
-* The union as `A ∪ B`
-* The set difference as `A \ B`
-* The symmetric difference as `A △ B`
-* The subset relation as `A ⊆ B`
-* The proper subset relation as `A ⊂ B`
+* The [subset] relation as `A ⊆ B` ("A is a subset of B")
+* The [proper subset] relation as `A ⊂ B` ("A is a subset of B, AND not A is not B")
+* The [intersection] as `A ∩ B`
+* The [union] as `A ∪ B`
+* The [set difference] as `A \ B`
+* The [symmetric difference] as `A △ B`
 
-For instance, the common members of `G` with respect to `H` can be denoted as:
 
+An important definition is the common members of `G` with respect to `H` denoted as:
 ```
 common(G,H) = members(G) ∩ members(nearest(G,H))
 ```
+> TODO - this definition doesn't take into account additions between `common(G, H)` and `H`
```

Comment:
This is what I'm confused by

Reply:
Yes, "common" is not "intersection". What this definition captures is the set of "remaining" members. If there is a bifurcation in the epochs, we need to go back to the common predecessor, look at its members, and see what *those* members have to do regarding the bifurcation. They are the ones who *witnessed* the bifurcation happening, and they have data that helps them decide which fork to select as the most preferred.

The reason this doesn't work as 
```
common(G,H) = members(G) ∩ members(H)
```

is because `common(G,H) = common(H,G)` is always true, which means rule 4.4. is always applied. Also `common(L,R) ⊂ common(R,L)` is never true, which means rule 4.5. is never applied. And `common(L,R) △ common(R,L)` is always empty, which means rule 4.6. is never applied.

So this means we only have rule 4.4., which says to **only** use tie-breaking rule to select the preferred epoch. This doesn't work in some cases, see e.g.

```mermaid
graph TB;
  zero[X: a,b,c,d]
  zero--"b excludes d"-->L[L: a,b,c]
  zero--"a excludes c,d\na adds e"-->R[R: a,b,e]
```

Suppose L is the tie-breaking winner. Then `a` and `b` decide to go to `L`, where `c` is, but `a` doesn't want `c` in the group! Do they have to re-remove `c`? And who performs that action? `a` or `b`? If it's both, then you may end up with forks again.

Reply:
done in another PR